third-party/zombienet/smoke_runtime_upgrade.zndsl

```diff
@@ -11,6 +11,8 @@ collator1: reports block height is greater than 5 within 200 seconds
 collator2: reports block height is greater than 5 within 200 seconds
 
 ## perform runtime upgrade
-collator1: parachain 1000 perform upgrade with ./runtime.compact.compressed.wasm within 200 seconds
+collator2: parachain 1000 perform upgrade with ./runtime.compact.compressed.wasm within 200 seconds
```

Comment:
This also needs to be done for collator1, right?

Reply:
In the previous iteration with the JS script, I performed the upgrade on collator 1 and tested propagation with the script on collator 2. Now I'm upgrading collator 2 because it's the second-latest client, so in client + runtime releases, it's the one most likely to fail. I think the upgrade is automatically propagated to 1, and after that, I'm making sure it keeps building blocks. However now, I am not 100% sure with different clients, WDYT?